File: src/main/java/ie/daithi/yeelight/model/Player.java
package ie.daithi.yeelight.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 *  {
 *      "local": true,
 *      "publicAddress": "200.200.200.200",
 *      "title": "Plex Web (Safari)",
 *      "uuid": "r6yfkdnfggbh2bdnvkffwbms"
 *  }
 */

@Data
@EqualsAndHashCode
public class Player {
	private Boolean local;
	private String publicAddress;
	private String title;
	private String uuid;
}

File: src/main/kotlin/ie/daithi/yeelight/model/Player.kt
package ie.daithi.yeelight.model

/**
 * {
 * "local": true,
 * "publicAddress": "200.200.200.200",
 * "title": "Plex Web (Safari)",
 * "uuid": "r6yfkdnfggbh2bdnvkffwbms"
 * }
 */
data class Player(
    val local: Boolean? = null,
    val publicAddress: String? = null,
    val title: String? = null,
    val uuid: String? = null
)